app/src/main/java/acr/browser/lightning/browser/bookmarks/BookmarksDrawerView.kt

```diff
@@ -78,6 +78,8 @@ class BookmarksDrawerView @JvmOverloads constructor(
             }
         }
 
+        iBinding.textTitle.setOnClickListener { }
```

Comment:
Surely this could be fixed by changing a flag in the XML layout.

Reply:
You're right, I just tried setting `android:clickable="true"` and it has the same effect

Reply:
It's weird though… Definitely go for the XML solution then I guess.